packages/cli-platform-apple/src/tools/pods.ts

```diff
@@ -61,10 +61,34 @@ export function generateMd5Hash(text: string) {
   return createHash('md5').update(text).digest('hex');
 }
 
-export function compareMd5Hashes(hash1: string, hash2: string) {
+export function compareMd5Hashes(hash1?: string, hash2?: string) {
   return hash1 === hash2;
 }
 
+async function getChecksum(podfileLockPath: string) {
+  const fileStream = fs.createReadStream(podfileLockPath);
+
+  const rl = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity,
+  });
+
+  let lines = [];
+  for await (const line of rl) {
+    lines.push(line);
+  }
```

Comment:
nit: if anyway you are reading the whole file in memory, and waiting for every line, why don't just

```suggestion
const file = fs.readFileSync(podfileLockPath);
```

?

Reply:
Ah, yea. When I was implementing it I thought about maybe streaming file from the end because `Podfile` contains checksum in the last few lines, but doing this with just `fs.readFileSync` is not probably a huge performance overhead.